main.tf

```
@@ -18,6 +18,7 @@ module "label" {
 resource "aws_ecr_repository" "default" {
   count = "${var.enabled == "true" ? 1 : 0}"
   name  = "${var.use_fullname == "true" ? module.label.id : module.label.name}"
+  tags  = "${var.tags}"
```

```suggestion
  tags  = "${module.label.id.tags}"
```

The `label` module will merge the provided tags with some generated tags (e.g. Name, NameSpace, Stage).
This is our pattern of naming and tagging resources.

---

Updated, thanks.